src/router.js

```diff
@@ -3,11 +3,12 @@ import Router from 'vue-router'
 import Signin from './views/Signin.vue'
 import Login from './views/Login.vue'
 import Home from './views/Home.vue'
+import BookList from './views/BookList.vue'
 
 Vue.use(Router)
 
 const privateRoutes = [{ path: '/signup', component: Signin }, { path: '/login', component: Login }]
-const publicRoutes = [{ path: '/home', component: Home }]
+const publicRoutes = [{ path: '/home', component: Home }, { path: '/books', component: BookList }]
```

Comment:
you can use lazy loading here, but it's ok this way

Reply:
Haven't seen that in the training but I'll look it up